use-cases.md

```diff
@@ -173,13 +174,13 @@ def traced_request(request, operation, http_client):
     # start a new span to represent the RPC
     span = tracer.start_span(
         operation_name=operation,
-        parent=parent_span,
+        parent=parent_span.context(),
         tags={'http.url': request.full_url}
     )
 
     # propagate the Span via HTTP request headers
     tracer.inject(
-        span,
+        span.context(),
```

Comment:
I think in Python we'd want context to be exposed as a `@property` (or even a stored attribute, but `@property` can make it read only), so the syntax would be `span.context`, not a function call.